packages/react-components/src/components/AutoComplete/AutoComplete.stories.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+
+import { StoryFn } from '@storybook/react';
+import debounce from 'lodash.debounce';
+
+import { StoryDescriptor } from '../../stories/components/StoryDescriptor';
+
+import { AutoComplete } from './AutoComplete';
+import { AutoCompleteProps } from './types';
+
+const placeholderText = 'Placeholder text';
+export default {
+  title: 'Forms/AutoComplete',
+  component: AutoComplete,
+  argTypes: {
+    onChange: {
+      description: 'The event handler for onChange',
+      action: 'changed',
+    },
+    icon: {
+      control: false,
+    },
+    readOnly: {
+      type: 'boolean',
+    },
+    description:
+      'Options that will be displayed in the picker. If they are strings, they will be converted to `IPickerListItem[]`',
+    table: {
+      type: {
+        summary: 'IAutoCompleteListItem[] | string[]',
+      },
+    },
+    alwaysShowAllOptions: {
+      description:
+        'If true, disables filtering of the options. Useful for getting options from an external source.',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: {
+        type: 'boolean',
+      },
+    },
+    autocompleteOpenOnInit: {
+      description:
+        'If true, the autocomplete list will be open when the component mounts.',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
+    single: {
+      description: 'If true, only shows the first matching item.',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+      control: {
+        type: 'boolean',
+      },
+    },
```

Comment:
Do we need these descriptions for props here? 🤔 
Are the comments on the interface not enough?

Reply:
They were necessary before, when AutoComplete was in the Input story, but that's no longer the case.